src/check_env.sh

```diff
@@ -22,12 +22,12 @@ command_exists docker-machine "$DOCKER_MACHINE_BIN" https://docs.docker.com/mach
 command_exists watchmedo "$WATCHMEDO_BIN" https://github.com/gorakhargosh/watchdog#installation
 command_exists rsync "$RSYNC_BIN" ""
 
-if ! (bash --version | grep "version 4" >/dev/null); then
-    echo "check bash(version 4)...\e[31mnot found\e[m"
+if ! (bash --version | grep -e "version \(4\|5\)" >/dev/null); then
```

Comment:
```suggestion
if ! (bash --version | grep -e "version [45]" >/dev/null); then
```
ちょっとうまく伝わってなかったですね。
正規表現としては `\(4|5\)` （-E の場合は `(4|5)` ）と書いても同じことを実現可能ですが、ある1文字の文字クラスにマッチさせるだけであれば `[45]` の方が表現が簡潔で好ましいと思います。（当初の指摘は `[4|5]` のように `|` を入れるのが誤りである、というだけで `[...]` を使わないで欲しい、というわけではないです）

`[...]` で表現できるのであればそれを使う、そうでなければ `\(...|...\)` を使う、というのが正規表現パターンを書く上での基本的な書き方になるかと。

Reply:
なるほど！確かに、そうですね！ご指摘ありがとうございます 🙇